src/runge-kutta.jl

```diff
@@ -0,0 +1,315 @@
+# This file contains the implementation of explicit Runkge-Kutta
+# solver from (Hairer & Wanner 1992 p.134, p.165-169).
+
+include("tableaus.jl")
+
+# intermediate level interface
+
+"""
+
+A general Runge-Kutta stepper (it cen represent either, a fixed step
+or an adaptive step algorithm).
+
+"""
+immutable RKStepper{Kind,tab_name,T<:Number} <: AbstractStepper
+    tableau::Tableau
+    function RKStepper()
+        tab = eval(tab_name)
```

Comment:
If you have an idea how to avoid it I'm all ears.  I thought about it and I couldn't come up with a better solution.